### PR TITLE
feat: add sample for updateJob

### PR DESCRIPTION
**Pull request by kelsk**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-scheduler/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #265  🦕

### CODE REVIEW

File: samples/updateJob.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  // const jobId = "JOB_ID"
+
+  // Construct the fully qualified location path.
+  const job = client.jobPath(projectId, locationId, jobId);
+
+  // Construct the request body.
+  const request = {
+    job: {
+      name: job,
+      description: 'Hello World example.',
```
Comment (averikitsch) [inline samples/updateJob.js:38]: Let's add a comment  for what's going on here in the request object

**Comment by averikitsch** [comment]
@kelsk When you have time, can we get this finished?
@bcoe how do we get the sample tests to run here?

**Comment by bcoe** [comment]
@averikitsch to kick off sample tests, you can add the `kokoro: force-run` label.

**Review by grant — APPROVED** [review]
Could not find issues with the sample.

**Pull request merged**